sites/next.skeleton.dev/src/content/docs/get-started/figma.mdx

```diff
@@ -1,26 +1,57 @@
 ---
 layout: '@layouts/LayoutDoc.astro'
 title: Figma
-description: A Figma UI Kit for Skeleton.
+description: The Figma UI Kit for Skeleton.
 order: 60
 ---
 
 export const components = componentSet;
 
-The Skeleton Figma UI Kit has been designed in coordination between [Bohdan Karplyak](https://www.linkedin.com/in/bohdan-karplyak-659a41162/), [Micha Mailänder](https://www.linkedin.com/in/micha-mail%C3%A4nder-548aba16a/), and [Skeleton Labs](https://www.skeletonlabs.co/).
+import FigmaBuyNow from '@components/docs/FigmaBuyNow.astro';
+import NavGrid from '@components/docs/NavGrid.astro';
 
-<img
-	src="https://github.com/skeletonlabs/skeleton/assets/1509726/956e6893-062e-4030-ba6a-d7b6e8acc4e8"
-	alt="figma-preview"
-	class="rounded-container"
-/>
+import { Image } from 'astro:assets';
+import ImgHero from '@images/get-started/figma/hero.webp';
+import ImgComponents from '@images/get-started/figma/components.webp';
+import ImgPlugin from '@images/get-started/figma/plugin.webp';
+import ImgTutorial from '@images/get-started/figma/tutorials.webp';
+
+<div class="relative">
+	<Image src={ImgHero} class="rounded-container" alt="hero" />
+	<a
+		class="btn btn-lg preset-filled-primary-500 flex items-center gap-2 whitespace-nowrap absolute top-6 right-6 shadow-lg w-48"
```

Comment:
That is interesting. What are your thoughts over having a neutral style button as a CTA? 
The CTA is the primary goal what the user should do on the page. In our case it's buying the Figma assets. So having a neutral style button (White and black) leads to no hierarchy differentiation between the CTA and the surrounding content. 
Companies that have optimized their CTA color contrast have seen substantial increases in conversion rates:
- Shopify reported a 12% increase in conversions after adjusting their CTA buttons for higher contrast.
- HubSpot experienced a 20% boost in user interactions by using bold orange against a white background.
- Booking.com improved accessibility and conversion rates by 15% through contrasting blue buttons on a white background

Reply:
![Screenshot 2024-11-13 at 9 48 15 AM](https://github.com/user-attachments/assets/8e7e8ffd-1471-4f3a-99cd-7affb67d886a)

It's not an issue of contrast here. It's an issue of assigning meaning. Neutral doesn't mean low contrast, in this case the "Get Started" button is still clearly visible, right? But we reserve color for even more critical items on a level above that. Such as the announcement, various alerts, etc.

It's similar to the comment I made the other day, we can't make every single action on the page the "most important" on the page. Otherwise nothing is important. Instead, think in terms of levels or grades.

- Most critical items - colored and really catch your eye
- Positive actions - ala CTAs, neutral, but still high contrast
- Neutral actions - the least contrast, using outlined currently

Reply:
From a design perspective we're also trying to keep a cohesive design. We need the Figma section to match the aesthetic of the rest of the app. If we decide to change the aesthetic, we'll do it across the board. Not at a one-off.

Reply:
Contrast is not only 100 vs 900. It can be also black-900 vs blue-500, and that's what I'm talking about. The most important objective on the Figma page is the conversion CTA. The whole page works towards that direction. Everything else is secondary.

Reply:
I do believe the Figma page is already fairly unique. Lots of big images, the "buy now" panel looks great, and I don't think anyone will have any trouble locating how to purchase if they are interested. But I'm happy to work with you on that once I have more bandwidth.

I have at least one more cosmetic pass in mind before launch. The homepage especially, which is currently a placeholder. We'll have something much more detailed, much informative and interactive, not unlike the v2 site (but hopefully even better). That can also include an additional call to action pointing at the Figma section.